app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -1,15 +1,17 @@
 <%
-  product_name ||= "Publishing"
-  public_environment = environment.eql?("public")
+  product_name ||= false
+  environment ||= false
+  environment_branding = !environment ? "public" : environment
```

Comment:
Does this branding have an impact if there isn't an environment? I also notice we don't have a class defined for `gem-c-layout-header--public` so I'm wondering if instead of this line we want to change line 14 to be `header_classes << "gem-c-layout-header--#{environment}" if environment`

Reply:
Great suggestion, have implemented, thanks.